### PR TITLE
Ui and fix bug

### PR DESCRIPTION
**Pull request by akoranos**

### CODE REVIEW

File: graphql/src/api/file/mutation.ts [hunk 1]
```diff
@@ -1,90 +1,82 @@
...
-        fs.unlinkSync(path.resolve(`./upload/${generateFile}`));
-        const data = {
-          fileName: generateFile,
-          path: `upload/${generateFile}`,
-          extension: mimetype,
-        };
-        return await ctx.prisma.file.create({
-          data: Object.assign(data),
-        });
+      resolve: async (_, { file, taskId }, ctx) => {
```
Comment (tonklaartisan) [inline graphql/src/api/file/mutation.ts:20]: Did I changed this?

Comment (akoranos) [reply graphql/src/api/file/mutation.ts]: Yes

File: web/src/components/LayoutDashboard.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ function LayoutDashboard({ children, noCard }: any) {
     <>
       <NavBar />
       <SideNav>
-        <Layout className="pt-8 pb-24 px-8 w-full">
+        <Layout className="pt-8 pb-24 px-8 w-full" style={{ fontFamily: 'Anuphan' }}>
```
Comment (tonklaartisan) [inline web/src/components/LayoutDashboard.tsx:12]: Define all font styles in `.css`

**Comment by tonklaartisan** [comment]
Use less inline style (`style={{}}`), move it to `.css`

File: web/src/App.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+@font-face {
```
Comment (tonklaartisan) [inline web/src/App.css:1]:
```
@font-face {
  font-family: Anuphan;
  src: url('./assets/fonts/Anuphan-Regular.woff') format('woff');
  font-weight: 400;
}
```

File: web/src/App.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (tonklaartisan) [inline web/src/App.css:1]: Move this to `public/assets/fonts/anuphan.css` with .woff fonts, then import into index.html inside `<head>`

**Review by tonklaartisan — APPROVED** [review]

**Pull request merged**